xzes40/transformer/src/main.cpp

```diff
@@ -19,23 +19,19 @@
 // Filename: xzes40.cpp
 // Description: Main entrypoint for the XZES40 Transformer application
 ///////////////////////////////////////////////////////////////////////////////
-
-// #include <xalanc/Include/PlatformDefinitions.hpp>
-// #include <xercesc/util/PlatformUtils.hpp>
-
-// #include <xalanc/XalanTransformer/XalanTransformer.hpp>
+//
+// NOTICE: THIS FILE HAS BEEN DEPRECATED IN FAVOR OF 'main.py'
+// IT REMAINS HERE FOR POSTERITY BUT IS NOT MAINTAINED AND SHOULD NOT BE LOOKED
+// AT AS A SOURCE OF TRUTH.
```

Comment:
So, main.py will be the major enter for our program?  Like the outside door for our program?